intranethome/templates/intranethome/mail_aliases.html

```diff
@@ -0,0 +1,178 @@
+        float: right;
+        margin-left: 5px;
+        }
+    
+      .active:after {
+        content: "\2212";
+        }
+    
+      .panel {
+        padding: 0 18px;
+        background-color: white;
+        overflow: hidden;
+        }
+</style>
+
+<body>
+
+{% extends "base/intranet_base.html" %}
+
+{% block body_class %}template-searchresults{% endblock %}
+
+{% block title %}Mail Aliases{% endblock %}
+
+{% block content %}
+
+<h1>Mail Aliases Page</h1>
+
+{%if error%}
+	There has been an error displaying the mailaliases page. Please contact <a href "mailto:intranet@lib.uchicago.edu">intranet@lib.uchicago.edu</a> and we will do our best to rectify the issue.
+{%else%}
+
+<button class="btn btn-default" onclick="expandAll()">Expand All</button>
+<button class="btn btn-default" onclick="contractAll()">Contract All</button>
+
+<h3>
+<a href="/mailaliases/">All</a> &nbsp; 
+<a href="/mailaliases/a/">A</a> &nbsp; 
+<a href="/mailaliases/b/">B</a> &nbsp; 
+<a href="/mailaliases/c/">C</a> &nbsp; 
+<a href="/mailaliases/d/">D</a> &nbsp; 
+<a href="/mailaliases/e/">E</a> &nbsp; 
+<a href="/mailaliases/f/">F</a> &nbsp; 
+<a href="/mailaliases/g/">G</a> &nbsp; 
+<a href="/mailaliases/h/">H</a> &nbsp; 
+<a href="/mailaliases/i/">I</a> &nbsp; 
+<a href="/mailaliases/j/">J</a> &nbsp; 
+<a href="/mailaliases/k/">K</a> &nbsp; 
+<a href="/mailaliases/l/">L</a> &nbsp; 
+<a href="/mailaliases/m/">M</a> &nbsp; 
+<a href="/mailaliases/n/">N</a> &nbsp; 
+<a href="/mailaliases/o/">O</a> &nbsp; 
+<a href="/mailaliases/p/">P</a> &nbsp; 
+<a href="/mailaliases/q/">Q</a> &nbsp; 
+<a href="/mailaliases/r/">R</a> &nbsp; 
+<a href="/mailaliases/s/">S</a> &nbsp; 
+<a href="/mailaliases/t/">T</a> &nbsp; 
+<a href="/mailaliases/u/">U</a> &nbsp; 
+<a href="/mailaliases/v/">V</a> &nbsp; 
+<a href="/mailaliases/w/">W</a> &nbsp; 
+<a href="/mailaliases/x/">X</a> &nbsp; 
+<a href="/mailaliases/y/">Y</a> &nbsp; 
+<a href="/mailaliases/z/">Z</a> &nbsp; 
+<a href="/mailaliases/number/">#</a>
+</h3>
+
+    {%for key, value in cleaned_data.items%}
+
+    <button class="accordion">{{key}}</button>
+    <div class="panel">
+      <table class="table">
+	<br>
+        {%for note_email in value%}
+          {%if note_email.note%}
+        	<tbody>
+		  <td> note: {{note_email.note}} </td>
+		</tbody>
+
+          {%elif note_email.triangle_brackets%}
+		  <tbody>
+                    <td><a href="mailto:{{note_email.triangle_brackets.1}}">{{note_email.triangle_brackets.0}}</a></td>
+                    <td>{{note_email.triangle_brackets.2}}</td>
+		  </tbody>
+
+          {%elif note_email.parentheses%}
+		  <tbody>
+                    <td><a href="mailto:{{note_email.parentheses.1}}">{{note_email.parentheses.0}}</a></td>
+                    <td>{{note_email.parentheses.2}}</td>
+		  </tbody>
+         
+	  {%elif note_email.plain_email%}
+		<tbody>
+		    <td><a href="mailto:{{note_email.plain_email}}">{{note_email.plain_email}}</a></td>
+		</tbody>
```

Comment:
Very nice the way the code enumerates the possible items that can appear in the JSON here.